strophe.h

```diff
@@ -332,9 +332,12 @@ int xmpp_stanza_to_text(xmpp_stanza_t *stanza,
 
 xmpp_stanza_t *xmpp_stanza_get_children(xmpp_stanza_t * const stanza);
 xmpp_stanza_t *xmpp_stanza_get_child_by_name(xmpp_stanza_t * const stanza, 
-					     const char * const name);
+        const char * const name);
 xmpp_stanza_t *xmpp_stanza_get_child_by_ns(xmpp_stanza_t * const stanza,
-					   const char * const ns);
+        const char * const ns);
+xmpp_stanza_t *xmpp_stanza_get_child_by_name_and_ns(xmpp_stanza_t * const stanza, 
+        const char * const name,
+        const char * const ns);
```

Comment:
Don't change coding style here. Functions arguments must be intended like this:
```C
void function(int arg1,
              int arg2,
              int arg3);
```

Reply:
Tabs and whitespaces are used in the same line in existing strophe.h. So so far it's not uniform.